tenzir/tenzir.cpp

```diff
@@ -219,6 +222,11 @@ auto main(int argc, char** argv) -> int {
       TENZIR_ERROR("could not load `tenzir.operators`: invalid record");
       return EXIT_FAILURE;
     }
+    auto force_tql2 = get_or(cfg, "tenzir.tql2", false);
```

Comment:
I think requiring all UDOs to be TQL2 already when the node is started in that mode is better, i.e., I favor your current implementation.

Reply:
It's the right choice for the node, but it also applies when running pipelines on the command-line where it can be kind of annoying if you don't have your UDOs upgraded yet. I personally don't mind, but still wanted to point it out.

Reply:
I think that's fine, given that generally the node is the main focus and the client process can mostly be considered a historic byproduct at this point in time.